datasets/us/ofac/enforcement_actions/crawler.py

```diff
@@ -0,0 +1,291 @@
+        clean_date = enforcement_date[0].strip().removesuffix(" -")
+        if not enforcements.within_max_age(context, clean_date, MAX_AGE_DAYS):
+            return True
+    return False
+
+
+def crawl_enforcement_action(context: Context, url: str) -> None:
+    article = context.fetch_html(url, cache_days=1)
+    article.make_links_absolute(context.data_url)
+    if article is None:
+        return
+    article_name = article.xpath(NAME_XPATH)[0]
+    article_content = article.xpath(CONTENT_XPATH)
+    assert len(article_content) == 1
+    article_element = article_content[0]
+    date = article.xpath(DATE_XPATH)[0]
+    article_html = tostring(article_element, pretty_print=True).decode("utf-8")
+    assert all([article_name, article_html, date]), "One or more fields are empty"
+
+    review = get_review(context, Defendants, article_name, MIN_MODEL_VERSION)
+    if review is None:
+        prompt_result = run_typed_text_prompt(context, PROMPT, article_html, Defendants)
+        review = request_review(
+            context,
+            url,
+            article_html,
+            HTML,
+            "Enforcement Action Notice",
+            url,
+            prompt_result,
+            MODEL_VERSION,
+        )
+
+    if check_something_changed(context, review, article_html, article_element):
+        # In the first iteration, we're being super conservative and rejecting
+        # export if the source content has changed regardless of whether the
+        # extraction result has changed. If we see this happening and we see that
+        # the extraction result reliably identifies real data changes, we can
+        # relax this to only reject if the extraction result has changed.
+
+        # Similarly if we see that broad markup changes don't trigger massive
+        # re-reviews but legitimate changes are reliably detected, we can allow
+        # it to automatically request re-reviews upon extraction changes.
+        global something_changed
+        something_changed = True
+        return
+
+    if not review.accepted:
+        return
+
+    for item in review.extracted_data.defendants:
+        entity = context.make(item.entity_schema)
+        entity.id = context.make_id(item.name, item.address, item.country)
+        entity.add("name", item.name, origin=DEFAULT_MODEL)
+        if item.address != item.country:
+            entity.add("address", item.address, origin=DEFAULT_MODEL)
+        entity.add("country", item.country, origin=DEFAULT_MODEL)
+        entity.add("alias", item.aliases, origin=DEFAULT_MODEL)
+        entity.add("topics", "reg.action")
+
+        # We try to link press releases that refer to an original press release number
+        # back to the original press release by using that in the sanction key.
+
+        # In practice often the entity ID and thus sanction ID differs because of
+        # different levels of address details in the press releases.
+        sanction = h.make_sanction(context, entity)
```

Comment:
we don't have any grouping key for this dataset so let's use the date to make sure notices about separate actions (and in fact all actions) are separate sanction entities. Let's update the comment accordingly.